docs/docs/ports.md

```diff
@@ -16,7 +16,7 @@ Port configuration for applications in Marathon can be confusing and there is [a
 
 *HOST networking*: `HOST` networking is used by non-Docker Marathon applications and Docker applications that use `HOST` mode networking. In this mode, applications bind directly to one or more ports on the host machine.
 
-*portMapping*: A _port mapping_ is necessary for Docker applications that use `BRIDGE` mode networking and is a tuple containing a host port, container port, service port and protocol. Multiple _port mappings_ may be specified for a Marathon application.
+*portMapping*: A _port mapping_ is used to define ports that should be considered as part of a resource offer. It is necessary for Docker applications that use `BRIDGE` mode networking and is a tuple containing a host port, container port, service port and protocol. Multiple _port mappings_ may be specified for a Marathon application.
```

Comment:
Even clearer:

A _port mapping_ is used to define ports that should be considered as part of a resource offer in `BRIDGE` mode. It is a tuple...
.
.
.
*ports*: The _ports_ array is used to define ports that should be considered as part of a resource offer in `HOST` mode....

I think most of the confusion comes from the use of the word `necessary`, sounds like `optional`.